.travis.yml

```diff
@@ -8,6 +10,8 @@ env:
   - SYSL_PLANTUML=http://www.plantuml.com/plantuml
 install:
   - pip install . pytest flake8
+  - pip install --upgrade six
```

Comment:
Both lines above

Reply:
i only added the "upgrade six" line because the travis docker image i was using had older version of six which generated an error when I tried to generate java code using reljam.

Not sure what you mean by "both lines". Anyway looks like the travis-ci does not have that issue.